### PR TITLE
Drag

### PR DESCRIPTION
**Pull request by ValentinGaudin**
The drag and drop take the skill from the table in BDD and not from a array in code.

### CODE REVIEW

File: src/Controller/LoginController.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            $infos = array_map('stripslashes', $infos);
+            $login = new LoginManager();
+            if ((($login->checkUser($infos)) == true) && ($login->checkPass($infos)) == true) {
+                echo "okay" ;
+            } else {
+                echo "not ok";
+            }
+                var_dump($_POST);
+                var_dump($infos);
+                //header('location:/welcome');
```
Comment (aldaitzwild) [inline src/Controller/LoginController.php:23]: No ! Leaving a var_dump or dead code (commented code) in a PR is forbidden

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
You have conflicts with dev and you have var_dump() and dead code in your PR

File: src/Controller/LoginController.php (hunk 1 above)
Comment (ValentinGaudin) [reply src/Controller/LoginController.php]: That's done with some other conflicts, no more dead code in LoginController. I paused the dev of the Login page. The var_dump() have been delete from the page.

File: src/Model/LoginManager.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        return $statement->fetch();
+    }
+
+    public function checkPass(array $pass)
+    {
+        $statement = $this->pdo->prepare("SELECT * FROM " . self::TABLE . " WHERE `pass` = :pass");
+        $statement->bindValue('pass', password_hash($pass['pass'], PASSWORD_BCRYPT), \PDO::PARAM_STR);
+        $statement->execute();
+        return $statement->fetch();
+    }
```
Comment (aldaitzwild) [inline src/Model/LoginManager.php:23]: No you don't check for login then for the password. 
If you do that you check if the login exists, if the password exists but not if this is the right password for the right login.

You have to check them both at the same time with the same query.

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Review by ValentinGaudin — COMMENTED** [review]
The Login manager and Login Html have been deleted, in fact we don't need this right now, pause the Login Page. And focus on the creation of a profil.

File: src/Model/LoginManager.php (hunk 2 above)
Comment (ValentinGaudin) [reply src/Model/LoginManager.php]: Stop the login, dev, put him in another branch of dev. DragAndDrop pull skill from a database and not in from scratch.

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/View/Login/inscription.html.twig [hunk 3]
```diff
@@ -1,7 +1,131 @@
...
+                    <div class=""> 
+                        {{ errors.town }} 
+                    </div>
+                </div>
+
+            <button type="submit" class="btn btn-outline-success col-12">Enregister</button>
+        </form>
+    </div>
+<<<<<<< HEAD
+=======
```
[the remark this answers is not in the document]
Comment (ValentinGaudin) [reply src/View/Login/inscription.html.twig]: Thanks, done !

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**